reporting/api/src/main/java/org/jboss/windup/reporting/rules/CreateApplicationListReportRuleProvider.java

```diff
@@ -70,12 +71,16 @@ private void createIndexReport(GraphContext context)
 
         GraphService<WindupVertexListModel> listService = new GraphService<>(context, WindupVertexListModel.class);
         WindupVertexListModel<ApplicationReportModel> applications = listService.create();
+        Map<String, WindupVertexFrame> relatedData = new HashMap<>();
         for (ApplicationReportModel applicationReportModel : applicationReportService.findAll())
         {
             if (applicationReportModel.isMainApplicationReport() != null && applicationReportModel.isMainApplicationReport())
+            {
                 applications.addItem(applicationReportModel);
+                if (ProjectService.SHARED_LIBS_UNIQUE_ID.equals(applicationReportModel.getProjectModel().getUniqueID()))
+                    relatedData.put("sharedLibsExists", applicationReportModel); // Instead of a boolean.
```

Comment:
Why not just use a boolean? The name is sort of odd for it not to be a boolean value.